src/SprykerEco/Zed/AmazonQuicksight/AmazonQuicksightConfig.php

```diff
@@ -18,6 +18,33 @@ class AmazonQuicksightConfig extends AbstractBundleConfig
      */
     protected const QUICKSIGHT_USER_REGISTER_NAMESPACE = 'default';
 
+    /**
+     * @var string
+     */
+    protected const QUICKSIGHT_USER_ROLE_READER = 'READER';
+
+    /**
+     * @var string
+     */
+    protected const QUICKSIGHT_USER_ROLE_AUTHOR = 'AUTHOR';
+
+    /**
+     * Specification:
+     * - Returns the list of available Quicksight user roles.
+     * - The list of available roles can be found here: {@link https://docs.aws.amazon.com/quicksight/latest/APIReference/API_User.html#QS-Type-User-Role}
+     *
+     * @api
+     *
+     * @return list<string>
+     */
+    public function getQuicksightUserRoles(): array
+    {
+        return [
+            static::QUICKSIGHT_USER_ROLE_READER,
+            static::QUICKSIGHT_USER_ROLE_AUTHOR,
+        ];
```

Comment:
I think 90% of developers will override the method to return what they need rather than redefine a constant. 
We already have many constants defined in this config and I don't think we will benefit from adding one more.